### ISSUE TITLE
Requiring Hanami::Utils::BasicObject causes TypeError (bind argument must be a subclass of Object)

### ISSUE THREAD

**Issue by deepj** [oracle/truffleruby#1710]
To reproduce

```
gem install 'hanami-utils'
ruby -r 'hanami-utils' -e 'require "hanami/utils/basic_object"'
```

Error:

```
~/.gem/truffleruby/2.6.2/gems/hanami-utils-1.3.2/lib/hanami/utils/basic_object.rb:45:in `define_method': bind argument must be a subclass of Object (TypeError)
	from ~/.gem/truffleruby/2.6.2/gems/hanami-utils-1.3.2/lib/hanami/utils/basic_object.rb:45:in `BasicObject'
	from ~/.gem/truffleruby/2.6.2/gems/hanami-utils-1.3.2/lib/hanami/utils/basic_object.rb:6:in `Utils'
	from ~/.gem/truffleruby/2.6.2/gems/hanami-utils-1.3.2/lib/hanami/utils/basic_object.rb:2:in `Hanami'
	from ~/.gem/truffleruby/2.6.2/gems/hanami-utils-1.3.2/lib/hanami/utils/basic_object.rb:1:in `<top (required)>'
	from ~/.rubies/truffleruby-19.1.0/lib/mri/rubygems/core_ext/kernel_require.rb:54:in `require'
	from ~/.rubies/truffleruby-19.1.0/lib/mri/rubygems/core_ext/kernel_require.rb:54:in `require'
	from -e:1:in `<main>'
```

It comes from https://github.com/hanami/utils/blob/master/lib/hanami/utils/basic_object.rb#L45

**Comment by deepj** [comment]
@chrisseaton Please, make this a priority

**Comment by deepj** [comment]
@eregon this seems to be resolved in 19.2.0

**Comment by eregon** [comment]
@deepj Yes indeed, I fixed this in f4af1b2c3207b366eb15b91f90fff903ac46e8c9 for 19.2.0 as marked in the ChangeLog but forgot to update this issue. Thanks for noticing and for the bug report.
